includes/classes/prod_cat_pulldown.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+	class prod_cat_pulldown extends base
+	{
+		var $pulldown, $attributes, $order_by;
+		/**
+		 * @var bool
+		 */
+		private $show_id;
+		/**
+		 * @var array
+		 */
+		private $exclude;
```

Comment:
Shouldn't there be handlers for one or both of these private variables? Otherwise the class has to be redefined again to take advantage of a change in the variable instead of simply reperforming the necessary step(s) with the adjusted value.

Reply:
i acknowledge that various vars used to control this class need work.  whether the addition of new config switches or session vars or something of the like would need to be addressed.  i did not want to get really built up into that infrastructure prior to getting feedback from the ZC team as to whether this approach will get approved or not.

Reply:
> prior to getting feedback from the ZC team as to whether this approach will get approved or not

Still queuing up a time to review code in depth.